core/src/calleeFunctions/helpers/wsteth.ts

```diff
@@ -0,0 +1,12 @@
+import BigNumber from '../../bignumber';
+import { getCollateralConfigBySymbol } from '../../constants/COLLATERALS';
+import { ETH_NUMBER_OF_DIGITS } from '../../constants/UNITS';
+import getContract from '../../contracts';
+
+export const convertWstethToSteth = async function (network: string, collateralAmount: BigNumber) {
```

Comment:
opinionated: it makes morse sense to me if the conversion functions are groupted together. otherwise being aware of conversion function existance becomes a harder task

Reply:
There is a conversion function per pool/per protocol. We might come with a smarter way later on